app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -61,7 +61,7 @@ message Bootstrap {
   }
 
   // Configuration for onboarding users in organizations with specific roles
-  repeated OnboardingSpec onboarding = 10;
+  repeated config.v1.OnboardingSpec onboarding = 10;
```

Comment:
out of curiosity, do you need to import or will be considered the same package?

Reply:
Yes, it's needed. I've tried removing the import and it doesn't work with and without `config.v1.` prefix. It seems they are not considered the same package after all.

Reply:
However, it works leaving the import and removing the prefix. That makes more sense to me. They are considered the same package, but it needs some help finding the new message definitions.